package.json

```diff
@@ -14,6 +14,7 @@
     "test": "jest --detectOpenHandles",
     "test:unit": "NODE_ENV=test npm run test 'test/unit/.*\\.test\\.ts'",
     "test:demo": "cd demo && npm run test:all",
+    "test:lock": "cd demo && npm run test:lock",
```

Comment:
Hi, I still failed in my local machine.

```
cd js-sdk/demo
npm install 
```

The error message:
![image](https://user-images.githubusercontent.com/26001097/196624283-60599208-2496-4a81-8854-a2bead3e8ac2.png)

Reply:
Hi, you can try it
`cd js-sdk
npm install `

Reply:
@LZHK1ng Then `npm install` succeeds but `npm run test:lock` fails:
![image](https://user-images.githubusercontent.com/26001097/196629814-86507796-48b1-4b18-a42d-fa0f14cc4dc8.png)

Reply:
Hi, The original approach worked in my machine.But it failed in CI😢.I try to use `yarn install` to fix it.
![image](https://user-images.githubusercontent.com/68213475/196685429-750efcad-88ee-41b1-9469-68f5326b592e.png)

Reply:
It still failed on my laptop:
```shell
cd demo
yarn install
```
![image](https://user-images.githubusercontent.com/26001097/196709025-63ddb155-82ac-42e1-9f7d-daff85fab695.png)